src/installer.ts

```diff
@@ -34,6 +34,11 @@ export async function getNode(
   let osPlat: string = os.platform();
   let osArch: string = translateArchToDistUrl(arch);
 
+  if (isLts(versionSpec)) {
+    core.warning('LTS version is provided. For LTS versions `check-latest` will be automatically set to true');
```

Comment:
No need to throw warning here because it produces a lot of noise in build results. `core.info` should be enough

Reply:
OK, will change.
For my learning, what do you mean by "a lot of noise"?

This 👇 looks okayish to me 🤔 

<img width="1113" alt="Screenshot 2021-06-21 at 11 45 08" src="https://user-images.githubusercontent.com/2134746/122742440-3c0f0500-d286-11eb-8d37-3a8749b57a40.png">

Reply:
Warning are also published in job summary that looks pretty annoying to me. (Like "something goes wrong" and I need to fix something). Especially, when customer runs job matrix, they will see a ton of such warnings.
![image](https://user-images.githubusercontent.com/16715858/122754129-e9404800-d29b-11eb-9842-dcf794b6030b.png)

Reply:
Thanks for the clarification! Now I see and agree, it's noisy 👍 

<img width="1089" alt="Screenshot 2021-06-21 at 14 24 18" src="https://user-images.githubusercontent.com/2134746/122761323-679cea00-d29c-11eb-9970-f384a54b04d4.png">

Reply:
I've fixed it for now. But I guess this code will go away anyway once we settle on `check-latest` usage.